frontend/src/pages/Activities/AddActivity/AddActivity.tsx

```diff
@@ -336,9 +336,10 @@ const AddActivity = ({}: AddActivityProps) => {
                     <div style={{ display: 'flex', flexDirection: 'column', gap: '5px' }}>
                         <DatePicker.RangePicker
                             showTime={{
-                                format: 'HH:mm',
+                                format: userSettings?.use24HrTime ? 'HH:mm' : 'hh:mm a',
                                 use12Hours: !(userSettings?.use24HrTime ?? true),
                             }}
+                            format={userSettings?.use24HrTime ? 'MM/dd/yyyy HH:mm' : 'MM/dd/yyyy hh:mm a' }
```

Comment:
i like this one slightly better, though it should match the format in activityDetail either way

Reply:
whoops forgot they were in 2 places.